nfabackend/src/main/resources-filtered/META-INF/persistence.xml

```diff
@@ -17,9 +17,12 @@
 			<property name="hibernate.show_sql" value="true"/>
 			<property name="hibernate.transaction.jta.platform" value="org.hibernate.service.jta.platform.internal.SunOneJtaPlatform"/>
 			<property name="javax.persistence.schema-generation.scripts.action" value="none"/>
-            <property name="javax.persistence.jdbc.user" value="${jdbc.user}"/>
```

Comment:
Are these lines part of the commit on purpose? Some Devs do not need / want them.